src/containers/other/PriceWidget/PriceWidget.tsx

```diff
@@ -0,0 +1,206 @@
+//- React Imports
+import { useCallback, useEffect, useRef, useState } from 'react';
+
+//- Style Imports
+import styles from './PriceWidget.module.scss';
+
+//- Icon Imports
+import openExternalUrl from 'assets/open-external-url.svg';
+
+//- Library Imports
+import useCurrency from 'lib/hooks/useCurrency';
+
+//- Constants Imports
+import { LABELS } from './constants';
+
+import { URLS } from 'constants/urls';
+
+//- Utils Imports
+import { WildUrlRowData, ZeroUrlRowData, Size, TokenType } from './utils';
+
+//- Global Utils Imports
+import { formatByDecimalPlace } from 'lib/utils/number';
+
+//- Component Imports
+import { HoverDropdown } from 'components';
+
+//- Selectors
+import * as selectors from './selectors';
+
+type PriceWidgetProps = {
+	isRoot: boolean;
+};
+
+const PriceWidget: React.FC<PriceWidgetProps> = ({ isRoot }) => {
+	//- Wallet Data
+	const {
+		wildPriceUsd,
+		wildPercentageChange,
+		zeroPriceUsd,
+		zeroPercentageChange,
+	} = useCurrency();
+
+	const isMounted = useRef<boolean>();
+
+	//- Token Data
+	const [data, setData] = useState<TokenType>();
+
+	const getTokenData = useCallback(() => {
+		if (isRoot) {
```

Comment:
Once there are more subdomains like wilder world these conditions will need to be updated to be more generic.